config/settings.yml

```diff
@@ -96,3 +96,4 @@ act_as_user_enabled: false
 
 reports:
   forms_api_forms_per_request_page: 100
+  organisation_to_exclude_from_live_report: null
```

Comment:
I think we'll need to make this setting accept a list of IDs so that we can exclude both the "GOV.UK Forms team" and "GOV.UK Forms End to end tests" organisations